picatic.io/app/components/example-container/example-container.template.html

```diff
@@ -0,0 +1,17 @@
+<figure ng-class="{'col-md-offset-2': $first}" class="grid__brick col-6 col-sm-6 col-md-4 col-xl-4 shuffle-item shuffle-item--visible" ng-repeat="example in $ctrl.examples" ng-cloak>
```

Comment:
`.col-*-*` classes should always be within a `.row` (or similar class) in order to get margins correct.
Ex:
```
<div class="container">
  <div class="row">
    <div class="col-xs-12 col-md-4"></div>
  </div>
</div>
```

Mostly speaking to this being a child of a `.col`, and its parent being a direct child of `.container`